is_core/generic_views/form_views.py

```diff
@@ -571,13 +574,38 @@ def get_success_url(self, obj):
         else:
             return self.request.get_full_path()
 
+    def _generate_export_types(self, export_types):
```

Comment:
odstraň vstupní parametr a používej self._get_export_types()

Reply:
to bych se zacyklil, prave v _get_export_types pouzivam to _generate_export_types